src/app/libmain.cxx

```diff
@@ -70,13 +70,13 @@ static RpmOstreeCommand commands[] = {
   { "reload", static_cast<RpmOstreeBuiltinFlags>(0),
     "Reload configuration",
     rpmostree_builtin_reload },
+  /* Defined in Rust */
   { "usroverlay", static_cast<RpmOstreeBuiltinFlags>(RPM_OSTREE_BUILTIN_FLAG_REQUIRES_ROOT),
-    "Apply a transient overlayfs to /usr",
-    rpmostree_builtin_usroverlay },
+    "Apply a transient overlayfs to /usr", NULL },
```

Comment:
Let's group all Rust-implemented commands like this together? I have a comment lower down I had added for this (where `module` lives). OK to not group too, though then let's delete that comment.

Reply:
Good catch, done!